app/models/goal.py

```diff
@@ -3,3 +3,16 @@
 
 class Goal(db.Model):
     goal_id = db.Column(db.Integer, primary_key=True)
+    title = db.Column(db.String)
```

Comment:
See the `Task` model for more information about required attributes being given `nullable=True`